packages/firebase_messaging/README.md

```diff
@@ -16,7 +16,10 @@ To use this plugin, add `firebase_messaging` as a [dependency in your pubspec.ya
 Check out the `example` directory for a sample app using Firebase Cloud Messaging.
 
 ### Android Integration
-
+Make sure you have created your Flutter app with the java option:
+```
+flutter create -i objc -a java appname
```

Comment:
Since Kotlin is now the default language for Android. This changes would mean it's not going to be easy for developers that already create their app and want to add `firebase_messaging` to get started. I believe Kotlin support is a must have at this moment since it's the default language for for new Flutter apps.

Reply:
Thanks a lot, i changed the code to kotlin 😀